apps/dynatrace/dynakube.yaml

```diff
@@ -67,10 +67,10 @@ spec:
 
   namespaceSelector:
     matchExpressions:
-      - key: kubernetes.io/metadata.name
+      - key: kubernetes.io/metadata.labels.ignored-by-dynatrace
```

Comment:
I think it might need to be just `ignored-by-dynatrace` but sure test it this way and edit if it doesn't work